doc/templates/components/edit-this-page.html

```diff
@@ -0,0 +1,32 @@
+{# Adapted from https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/components/edit-this-page.html #}
+
+{% extends "basic-ng/components/edit-this-page.html" %}
+{% from "basic-ng/components/edit-this-page.html" import determine_page_edit_link with context %}
+
+{%- macro furo_edit_button(url) -%}
+<!-- {%- if "auto_examples" in url -%}
+{{ url }}
+{%- endif -%} -->
+<div class="edit-this-page">
+  <a class="muted-link" href="{{ url|replace("doc/auto_examples", "examples")|replace(".rst", ".py") }}" title="{{ _("Edit this page") }}">
```

Comment:
This is pretty much the only line that was changed compared to the original furo component.

Reply:
Pronably worth adding a comment in this file explaining why it's there.

Reply:
> Pronably worth adding a comment in this file explaining why it's there.

Done